### PR TITLE
[Feature] add NavBar component

### PR DESCRIPTION
**Pull request by jomadoye**
This PR adds the Navigation component
A screen shot is attached below.
screen shot 2017-07-31 at 2 15 50 am

### CODE REVIEW

**Comment by BolajiOlajide** [comment]
@kingisaac95 Have you reached a final decision on the color scheme to use?
I need a response before this can be merged.

File: client/components/NavigationBar/NavigationBar.jsx
Comment (BolajiOlajide) [inline client/components/NavigationBar/NavigationBar.jsx]: can the idea be lower case?

Comment (BolajiOlajide) [reply client/components/NavigationBar/NavigationBar.jsx]: they can be called `navbar-text`

File: client/components/NavigationBar/NavigationBar.test.js
Comment (BolajiOlajide) [inline client/components/NavigationBar/NavigationBar.test.js]: Kindly remove this comment since it's not needed

File: client/index.html
Comment (BolajiOlajide) [inline client/index.html:10]: can you do us the favor of importing `jquery` too?

**Review by BolajiOlajide — COMMENTED** [review]
Good work Jed!

**Comment by andela-oakinseye** [comment]
Good job Jed!

File: client/components/NavigationBar/NavigationBar.jsx
Comment (kingisaac95) [inline client/components/NavigationBar/NavigationBar.jsx]: I notice you're using &nbsp; and a space in the brand logo. Please refactor it and use padding instead for consistency.

File: client/index.html
Comment (kingisaac95) [reply client/index.html:10]: I have imported it locally. It'll reflect with my PR. Is that okay with you @andela-jomadoye?

**Comment by jomadoye** [comment]
@BolajiOlajide All changes have been made.

**Review by BolajiOlajide — APPROVED** [review]
This looks good and ready to be merged.

**Pull request merged**
